projects/ngneat/transloco-messageformat/src/lib/messageformat.config.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+
+import * as MessageFormat from 'messageformat';
+
+export const TRANSLOCO_MESSAGE_FORMAT_CONFIG = new InjectionToken<MessageformatConfig>(
+  'TRANSLOCO_MESSAGE_FORMAT_CONFIG'
+);
+
+export interface MessageformatConfig extends MessageFormat.Options {
+  locales?: { [locale: string]: Function } | string[] | string;
```

Comment:
It doesn't already come from `MessageFormat.Options`?

Reply:
No, It's the second parameter of the constructor